masks/mask_class/mask_class.py

```diff
@@ -0,0 +1,106 @@
+import numpy as np
+import cv2 as cv
+from math import sqrt
+from sympy.ntheory import isprime, quadratic_residues
+from scipy.signal import max_len_seq
+from perlin_numpy import generate_perlin_noise_2d
+
+
+class Mask():
+
+    def __init__(self, sensor_size_px: tuple, 
+                 sensor_size_m: tuple, 
+                 feature_size: tuple, 
+                 distance_sensor: float) -> None:
+        
+        self.mask = None
+        self.sensor_size_px = sensor_size_px
+        self.sensor_size_m = sensor_size_m
+        self.feature_size = feature_size
+        self.distance_sensor = distance_sensor
+    
+    def shape(self):
+        return self.mask.shape
+
+
+
+class FlatCam_Mask(Mask):
```

Comment:
Rename to e.g. `CodedAperture` and link to paper: https://ieeexplore.ieee.org/document/7517296
and describe process to generate mask

Reply:
And for FlatCam_Mask and FZA_Mask ?